app/cdap/components/WrangleHome/Components/OngoingDataExplorations/Constants.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import T from 'i18n-react';
+
+export const CONNECTOR_TYPE = T.translate(
+  'features.WranglerNewUI.OnGoingDataExplorations.labels.connectorType'
+).toString();
```

Comment:
Why call `toString` after each `translate`?

Reply:
removed these constants as we are not showing these constants on UI